content/en/docs/Writing policies/match-exclude.md

```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
 * `networking.k8s.io/v1/NetworkPolicy`
 * `crd.antrea.io/v1alpha1/NetworkPolicy`
 
+Wildcards supported formats:
+
+* `Group/*/Kind`
+* `*/Kind`
+* `*`
+
+{{% alert title="Note" color="info" %}}
+* A policy using wildcards in `match` or `exclude` is not allowed in background mode.
+* A policy using wildcards does not `generate` or `verifyImages` rule types, and does not support `forEach` declarations.
```

Comment:
Sorry, must have messed up:

"A policy using wildcards does not support `generate` or `verifyImages` rule types, and does not support `forEach` declarations."